### ISSUE TITLE
[Linux] Default a11y role on Linux is frame, which is for a toplevel window

### ISSUE THREAD

**Issue by robert-ancell** [flutter/flutter#98897]
## Steps to Reproduce

1. Execute `flutter run` on the code sample
2. Open accerciser.
3. Browse the objects.

**Expected results:**

The role of the a11y elements from Flutter should have an appropriate role.

**Actual results:**

The elements all have the "frame" role. This role is for [toplevel windows](https://docs.gtk.org/atk/enum.Role.html#frame), the "panel" role seems more appropriate (as used in a11y elements from GTK).

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: const MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  const MyHomePage({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            const Text(
              'One',
            ),
            const Text(
              'Two',
            ),
            const Text(
              'Three',
            ),
          ],
        ),
      ),
    );
  }
}
```

</details>

**Comment by maheshmnj** [comment]
Hi @robert-ancell, thanks for filing the issue. I am able to reproduce the issue on stable and the master channel. All three text widgets have frame role

![image](https://user-images.githubusercontent.com/31410839/155112220-e1f83c54-11f1-4cbe-beea-dd40f257aa66.png)
 

<details>
<summary>flutter doctor -v</summary>


```
[✓] Flutter (Channel stable, 2.10.2, on Ubuntu 20.04.3 LTS 5.13.0-28-generic, locale en_GB.UTF-8)
    • Flutter version 2.10.2 at /home/mahesh/Development/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 097d3313d8 (3 days ago), 2022-02-18 19:33:08 -0600
    • Engine revision a83ed0e5e3
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/mahesh/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio
    • Flutter plugin version 63.2.1
    • Dart plugin version 203.8452
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.63.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.34.0

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 20.04.3 LTS 5.13.0-28-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 97.0.4692.99

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
```
✓] Flutter (Channel master, 2.11.0-0.0.pre.633, on Ubuntu 20.04.3 LTS 5.13.0-28-generic, locale en_GB.UTF-8)
    • Flutter version 2.11.0-0.0.pre.633 at /home/mahesh/Development/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 54a9d65282 (3 days ago), 2022-02-19 10:09:26 -0800
    • Engine revision f6ea6186ee
    • Dart version 2.17.0 (build 2.17.0-125.0.dev)
    • DevTools version 2.11.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/mahesh/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/mahesh/Downloads/android-studio-2020.3.1.25-linux/android-studio
    • Flutter plugin version 63.2.1
    • Dart plugin version 203.8452
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.63.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.34.0

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Ubuntu 20.04.3 LTS 5.13.0-28-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 97.0.4692.99

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>
